boa/src/object/gcobject.rs

```diff
@@ -211,111 +316,10 @@ impl GcObject {
     pub fn construct(
```

Comment:
This should improve performance a bit on debug builds, since it's a 1 liner.

```suggestion
    #[inline]
    pub fn construct(
```